Projections/Test/Test.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <GenerateTestProjection Condition="'$(GenerateTestProjection)$(Configuration)' == 'Release'">true</GenerateTestProjection>
+    <GenerateTestProjection>true</GenerateTestProjection>
```

Comment:
I think committing this change may have been unintentional?

Reply:
the test projection executes quickly, even with the debug cswinrt.exe

Reply:
Ok! Just wanted to make sure.